infrastructure/deployment/deploy.sh

```
@@ -355,10 +355,8 @@ export ROTATING_APM_ELASTIC_PASSWORD=`generate_password`
 
 # Download core compose files to /tmp/
 for compose_file in ${COMPOSE_FILES_DOWNLOADED_FROM_CORE[@]}; do
-  if [ ! -f $compose_file ]; then
     echo "Downloading $compose_file from https://raw.githubusercontent.com/opencrvs/opencrvs-core/$VERSION/$(basename $compose_file)"
-    curl -o $compose_file https://raw.githubusercontent.com/opencrvs/opencrvs-core/$VERSION/$(basename $compose_file)
-  fi
+    curl --fail -o $compose_file https://raw.githubusercontent.com/opencrvs/opencrvs-core/$VERSION/$(basename $compose_file)
```

Is there a specific reason to fail silently here using`--fail`?